### ISSUE TITLE
deviation_inverse_freq_curve

### ISSUE THREAD

**Issue by l-omar-chehab** [mne-tools/mne-features#34]
@agramfort 

Should we include a deviation to the inverse freq curve (prototypical power spectral density) functionality?

If so, do we do a linear regression on the log-log curve, in which case our estimator is formally:
estimated power = k1 / f^lambda , where k1 = exp(estimated intercept) and lambda = - estimated slope

Or, as in the Winkler 2011 paper, do we add a term k2 to our estimator such that: 
estimated power = (k1 / f^lambda) - k2 

In other words, do we assume or not that the power converges to 0 for very high frequencies?

**Comment by jbschiratti** [comment]
Thank you @l-omar-chehab for opening this issue. 

Would it be used as a feature (MSE between the PSD and the 1/f^c curve, for example?) or to do "spectral normalization" (as in [Char2007](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2235870/)) before feature extraction?

**Comment by l-omar-chehab** [comment]
Thank you for contributing :)

For my part, I'm viewing it as a feature (typically the MSE that you mentioned), but I do not necessarily have enough perspective to know how it may be of interest to many. 

If you'd like to weigh in on that @jbschiratti @agramfort  ?

**Comment by agramfort** [comment]
I think the idea here is to estimate a feature.

**Comment by jbschiratti** [comment]
Ok ! @l-omar-chehab Feel free to open a PR if you have some code ready.

**Comment by jbschiratti** [comment]
See #37.
